content/_index.md

```diff
@@ -18,9 +18,17 @@ Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an
 {{< /home/section >}}
 
 {{< home/section title="Why?" >}}
-By correlating every software [artifact](/glossary/artifact) to a verifiable, complete, and concise [artifact tree](/glossary/artifact_tree), GitBOM enables run-time detection of potential vulnerabilities, regardless of the depth in a depencency tree from which that vulnerability originated.
+By constructing a complete, concise, and verifiable [artifact tree](/glossary/artifact_tree) for every [software artifact](/glossary/artifact), [GitBOM](/glossary/gitbom) enables:
+- Run-time detection of potential vulnerabilities, regardless of the depth in a dependency tree from which that vulnerability originated
+- Identifying license obligations across both open source and commercial software
+- More reliable attestation
```

Comment:
gitbom does not perform, nor provide any metadata salient to, remote attestation.

again, I deleted this line to avoid inviting debate about what we mean by "attestation" here -- since i do not believe this means the same thing as the definition proposed within the IETF RATS WG.

Reply:
Same comment as above, will delete.